packaging/hudi-hadoop-mr-bundle/pom.xml

```diff
@@ -67,8 +67,9 @@
                 <includes>
                   <include>org.apache.hudi:hudi-common</include>
                   <include>org.apache.hudi:hudi-hadoop-mr</include>
-
+                  <!-- Parquet -->
                   <include>org.apache.parquet:parquet-avro</include>
+                  <include>org.apache.parquet:parquet-hadoop-bundle</include>
```

Comment:
This change means that we are now bundling parquet for hive. not  just `parquet-avro`. Should we fix the Mr bundle's parquet-avro version alone instead

Reply:
There are two proposals：
1）The read and write engines (according to spark parquet-avro version)use the same version which is the meaning of this patch
2）The read engines such as hive use their own parquet-* version

for hive2 parquet-hadoop version is 1.8.1,  hive3 parquet-hadoop version is 1.10.0, which is not compatible with the version of parquet-avro. 

So  the second solution may take 1.8.1 for hive2, 1.10.0 for hive3 ,but has also a bit inconsistent with the write engines.

Reply:
@RexXiong if we enforce parquet-avro version using a different variable say `hive.parquet.version` to always `1.10.x`, and shade it in hadoop-mr-bundle, this should be ok ?  Would parquet-avro 1.10.x work with parquet-hadoop 1.8.1 ? if so, we don't have to create hive profiles.

Reply:
@xushiyan test seems  parquet-avro 1.10.x is compatible with parquet-hadoop 1.8.1. So I will specify the parquet version of parquet-avro for hadoop-mr-bundle, and this solution was also the first proposed.